src/tools/wasm-shell.cpp

```diff
@@ -315,7 +315,10 @@ int main(int argc, const char* argv[]) {
         if (!valid) {
           WasmPrinter::printModule(modules[moduleName].get());
         }
-        assert(valid);
+        if (!valid) {
+          std::cerr << "Invalid module!\n";
+          abort();
```

Comment:
Actually I guess I just said we shouldn't use abort for this purpose. The reason is that it's treated by the system more like a crash (e.g. you get a core file on Linux, and the exit status is different). Did we have a `Fatal` class somewhere for this purpose at one point?